de.dlr.sc.virsat.external.lib.swagger/pom.xml

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and 
+	Software Technology, Germany. This program and the accompanying materials 
+	are made available under the terms of the Eclipse Public License 2.0 which 
+	is available at http://www.eclipse.org/legal/epl-2.0. SPDX-License-Identifier: 
+	EPL-2.0 -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>de.dlr.sc.virsat.external.lib.maven.parent</artifactId>
+		<groupId>de.dlr.sc.virsat</groupId>
+		<version>4.12.1-SNAPSHOT</version>
+		<relativePath>../de.dlr.sc.virsat.external.lib/maven/pom.xml</relativePath>
+	</parent>
+
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>de.dlr.sc.virsat.external.lib.swagger</artifactId>
+	<name>de.dlr.sc.virsat.external.lib.swagger</name>
+	<packaging>eclipse-plugin</packaging>
+
+	<!-- Execute with profile downloadLib which is defined in the parent pom 
+		to update all libs and licenses from maven -->
+
+	<properties>
+		<swagger.version>1.6.2</swagger.version>
+	</properties>
+
+	<dependencies>
```

Comment:
We now wrap these into the downloadLibs profile (check other external libs) to prevents maven from downloading these dependencies during build time.